### ISSUE TITLE
Livestream URL not properly reconized as livestream

### ISSUE THREAD

**Issue by amina013** [corbindavenport/noplugin#48]
I can't play the video on a site. I have pasted the Url here. http://www.vutube.edu.pk/vu-channels/vu-channel-vtv-1

**Comment by corbindavenport** [comment]
Looks like the embed is trying to play a livestream, and NoPlugin doesn't recognize it as a livestream:
```
[NoPlugin] Replaced plugin embed for http://111.68.103.51:1234
```
I'll have to implement some kind of check for URLs without a file ending.

**Comment by corbindavenport** [comment]
A fix for this is now in the `compat-fixes` branch. Once I'm positive this won't break something else, I'll merge it into master and release an extension update.

**Comment by corbindavenport** [comment]
The fix is now rolling out as part of NoPlugin 5.1.
